web/ui.js

```diff
@@ -739,7 +739,65 @@ $(function() {
                 var tradeCounts = data['trades'];
                 var useCounts = data['uses'];
 
-                var htmlData = "---------- <span title=\"Currency\">BTC</span> -----------\n";
+                var htmlData = "----- <span title=\"Total Portfolio\">TOTAL PORTFOLIO</span> -----\n";
+
+                // sum up altcoin totals
+                var altcoinTotals = {};
+                Object.keys(wallets).forEach(function(coin) {
+                    Object.keys(wallets[coin]).forEach(function(xid) {
+                        if (xid in altcoinTotals) {
+                            altcoinTotals[xid]['balance'] += wallets[coin][xid]['balance_BTC'];
+                            altcoinTotals[xid]['change'] += wallets[coin][xid]['change_BTC'];
+                        } else {
+                            altcoinTotals[xid] = {}
+                            altcoinTotals[xid]['balance'] = wallets[coin][xid]['balance_BTC'];
+                            altcoinTotals[xid]['change'] = wallets[coin][xid]['change_BTC'];
+                        }
+                    });
+                }); 
+                var altcoinBalance = 0;
+                var altcoinChange = 0;
+                Object.keys(altcoinTotals).forEach(function(xid) {
+                    var exname = no2el(xid);
+                    while (exname.length < 5) {
+                        exname = " " + exname;
+                    }
+                    
+                    htmlData += exname;
+                    htmlData += ": ";
+                    htmlData += "<a href=\"#\" coin=\"alt_btc\" exchange=\"" + xid + "\" mode=\"0\" class=\"showGraph\" title=\"Total balance (BTC + Altcoins)\">";
```

Comment:
Can you please avoid making these links in this PR, since on its own, when this PR gets merged, clicking on these links won't do anything?